coq/coq.el

```diff
@@ -722,6 +722,7 @@ If locked span already has a state number, then do nothing. Also updates
   "Last goal that Emacs looked at.")
 
 (defun coq-goal-hyp ()
+  "Instanciation for ``pg-topterm-goalhyplit-fn'', see there."
```

Comment:
Hi @hendriktews ! Thanks. Just a question: what's the semantics of <code>\`\`function-name''</code>? w.r.t. <code>`function-name'</code> with single quotes?

Reply:
Fixed now.